Context超时及超时传递相关文档

Hi,

我这边使用过程中，有发现，如果实现了一个API，API背后做了一个耗时较长的调用，超时失败后，再去调用`dao`来查询数据时，会报：

```
ERROR 08/09-17:21:47.017 /Users/jacky/Colstuwjx/Projects/Engineering/kpaas/internal/core/engine/workflow/workflow.go:121 error trans start for ticket 3, Get https://192.168.99.108:8443/api/v1/namespaces/default/pods: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
ERROR 08/09-17:21:47.024 /Users/jacky/Colstuwjx/Projects/Engineering/kpaas/internal/server/http/release_ticket.go:136 error get release ticket with steps by id 3 after did start, query:SELECT id,ctime,mtime,strategy,current_step,current_rolling_type,step_timeout,status,operator,app_cluster_id,incumbent_id,contender_id FROM release_ticket WHERE (id=?), args:[3]: context deadline exceeded
ERROR 08/09-17:21:47.024 http-access-log ip=127.0.0.1 params= err=error get release ticket with steps by id 3 after did start, query:SELECT id,ctime,mtime,strategy,current_step,current_rolling_type,step_timeout,status,operator,app_cluster_id,incumbent_id,contender_id FROM release_ticket WHERE (id=?), args:[3]: context deadline exceeded ts=5.071452207 method=PUT path=/api/v1/release_tickets/3/start ret=-500 msg=-500 stack=error get release ticket with steps by id 3 after did start, query:SELECT id,ctime,mtime,strategy,current_step,current_rolling_type,step_timeout,status,operator,app_cluster_id,incumbent_id,contender_id FROM release_ticket WHERE (id=?), args:[3]: context deadline exceeded traceid= user=no_user timeout_quota=0.999893903
```

有看到仓库代码有关于[超时传递](https://github.com/bilibili/kratos/blob/master/pkg/time/time.go#L50)的处理，但是我这边设置了最上层的context为30s超时，耗时长调用为5s超时，随后调用`dao`获取数据仍然是上面的错误日志。

烦请帮忙确认一下问题所在，或补充一下相关的文档，谢谢！

--- a/doc/wiki-cn/blademaster-mod.md
+++ b/doc/wiki-cn/blademaster-mod.md
@@ -26,6 +26,7 @@ type Context struct {
 ```
 
 * 首先可以看到 blademaster 的 Context 结构体中会 embed 一个标准库中的 Context 实例，bm 中的 Context 也是直接通过该实例来实现标准库中的 Context 接口。
+* blademaster 会使用配置的 server timeout (默认1s) 作为一次请求整个过程中的超时时间，使用该context调用dao做数据库、缓存操作查询时均会将该超时时间传递下去，一旦抵达deadline，后续相关操作均会返回`context deadline exceeded`。
 * Request 和 Writer 字段用于获取当前请求的与输出响应。
 * index 和 handlers 用于 handler 的流程控制；handlers 中存储了当前请求需要执行的所有 handler，index 用于标记当前正在执行的 handler 的索引位。
 * Keys 用于在 handler 之间传递一些额外的信息。

